playwright/sync_api/__init__.py

```diff
@@ -74,9 +74,7 @@ def sync_playwright() -> PlaywrightContextManager:
 
 
 __all__ = [
-    "async_playwright",
```

Comment:
What is the reasoning that this gets removed?

Reply:
`async_playwright` should not be included in the sync package